### PR TITLE
DIRECTOR: Add detection for Arnie and its demo

### PR DESCRIPTION
**Pull request by VelocityRa**
My versions are in Swedish but I'm pretty sure there aren't any other versions, that's why I chose `Common::EN_ANY`.

The game doesn't currently boot as `loadCastData` eventually tries to read non-Cast data and asserts [here](https://github.com/scummvm/scummvm/blob/master/engines/director/score.cpp#L495).

### CODE REVIEW

File: engines/director/detection_tables.h [hunk 1]
```diff
@@ -527,6 +527,34 @@ static const DirectorGameDescription gameDescriptions[] = {
 		2
 	},
 
+	{
+		{
+			"ernie",
+			"Ernie - Demo",
```
Comment (sev-) [inline engines/director/detection_tables.h]: Since you're adding another target, you need to register it in the detection.cpp file as well.

**Comment by sev-** [comment]
D5 support is quite far away in current codebase. We haven't even looked much into booting it.

**Comment by VelocityRa** [comment]
Ah, alright. Issue is fixed anyway.

**Comment by sev-** [comment]
Thanks

**Comment by henke37** [comment]
English versions existing or not, the language is Swedish and nothing else.

**Pull request merged**